test/verify/check-networking-bridge

```diff
@@ -40,7 +40,7 @@ class TestNetworking(NetworkCase):
         # connected to, and the bridge we are creating here.
```

Comment:
This comment needs to be updated, or removed.

Reply:
I thought about it,  but it's still not wrong -- the two interfaces will continue to be independent, just that this isn't a special case really. They would be on the same network if the same VirtNetwork() object was passed to them. I'm still happy to remove it if you prefer, though.

Reply:
Yes, hmm.  The comment is written in a style that explains an explicit thing in the code. "...is connected ... to avoid ...".  This is not valid anymore.  It should be "Note that all additional interfaces are each connected to their own separate network by default.  Thus, there is no loop between ...."

Sorry to be pedantic, but comment rot needs to be stopped! :-)

Reply:
No no, that's good! Pushed.

Reply:
Thanks!